<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton9">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton0">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonD">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMul">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonM">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonP">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>142</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>16</number>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Dec</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCalc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonDec">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Decimal</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonBin">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Binary</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonHex">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Hexadecimal</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>332</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
